plugins/space-configure/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@flatfile/plugin-space-configure",
+  "version": "0.0.0",
+  "description": "A plugin for configuring a Flatfile Space.",
+  "registryMetadata": {
+    "category": "configure"
```

Comment:
@ashleygmulligan2 / @bangarang  do you have a preference for a category here?

Reply:
let's do core/